styles/globals.css

```diff
@@ -15,24 +15,36 @@
 
   body,
   #__next {
-    @apply flex min-h-full flex-col text-base leading-relaxed;
+    @apply flex min-h-full flex-col text-base text-uog-color-body-copy leading-relaxed;
```

Comment:
This might be too global. Other elements seem to be inheriting the colour (e.g., a element)

![image](https://github.com/user-attachments/assets/40467040-023f-4cc7-a631-68a929796c3a)

Reply:
Resolved